librz/core/cmd.c

```diff
@@ -2163,6 +2157,59 @@ RZ_API int rz_core_cmd_pipe(RzCore *core, char *rizin_cmd, char *shell_cmd) {
 	return ret;
 }
 
+static char *system_exec_stdin(int argc, char **argv, const char *input) {
+	char *output = NULL;
+	if (!rz_subprocess_init()) {
+		RZ_LOG_ERROR("Cannot initialize subprocess.\n");
+		return NULL;
+	}
+
+	RzSubprocessOpt opt = {
+		.file = argv[0],
+		.args = (const char **)&argv[1],
+		.args_size = argc - 1,
+		.envvars = NULL,
+		.envvals = NULL,
+		.env_size = 0,
+		.stdin_pipe = RZ_SUBPROCESS_PIPE_CREATE,
+		.stdout_pipe = RZ_SUBPROCESS_PIPE_CREATE,
+		.stderr_pipe = RZ_SUBPROCESS_PIPE_STDOUT,
+	};
+
+	RzSubprocess *proc = rz_subprocess_start_opt(&opt);
+	if (!proc) {
+		RZ_LOG_ERROR("Cannot start subprocess.\n");
+		rz_subprocess_fini();
+		return NULL;
+	}
+
+	size_t input_len = strlen(input);
+	rz_subprocess_stdin_write(proc, (const ut8 *)input, input_len);
+	rz_subprocess_wait(proc, UT64_MAX);
+
+	output = rz_subprocess_out(proc);
+	rz_subprocess_free(proc);
+	rz_subprocess_fini();
+
+	return output;
+}
+
+RZ_API RzCmdStatus rz_core_cmd_pipe(RzCore *core, char *rizin_cmd, int argc, char **argv) {
```

Comment:
Since this is new API function, add Doxygen please.

Reply:
not sure if we should support this as an API.

Reply:
Probably not.

Reply:
please anyway a small description would be good IMHO.